components/movementsensor/client_test.go

```diff
@@ -152,6 +158,12 @@ func TestClient(t *testing.T) {
 		test.That(t, acc1, test.ShouldResemble, acc)
 		test.That(t, injectMovementSensor.AccuracyFuncExtraCap, test.ShouldResemble, map[string]interface{}{"foo": "bar"})
 
+		la, err := gps1Client.LinearAcceleration(context.Background(), map[string]interface{}{"foo": "bar"})
+		logger.Info(err)
+		test.That(t, err, test.ShouldBeNil)
+		test.That(t, la, test.ShouldResemble, la)
```

Comment:
it appears this is checking something against itself, did you mean to do this?

Reply:
Thanks for catching that!